.github/workflows/cmake.yaml

```diff
@@ -29,19 +29,19 @@ jobs:
       - name: Install ninja (apt)
         run: sudo apt-get install -y ninja-build
         if: runner.os == 'Linux'
+      - name: Set up MSVC (Windows)
+        uses: ilammy/msvc-dev-cmd@0b201ec74fa43914dc39ae48a89fd1d8cb592756 # v1.13.0
```

Comment:
Does this step need an `if: runner.os == 'Windows'` line?

Reply:
Not really necessary, this action doesn't do anything outside of Windows.